Vagrantfile

```diff
@@ -36,7 +36,14 @@ Vagrant.configure("2") do |config|
 		cd /vagrant
 		poetry install
 		nohup poetry run flask run --host 0.0.0.0 > logs.txt 2>&1 &
```

Comment:
Did you mean to get rid of this line?

You are currently running both flask and gunicorn.